src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,86 @@
+package storage;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import exceptions.DukeException;
+import tasklist.TaskList;
+
+
+/**
+ * Class representing disk storage used in Duke
+ * to backup task list.
+ * Requires valid file parh to be set at
+ * each run of Duke.
+ */
+public class Storage {
+    private Path path;
+    private final StorageReader storageReader;
+    private final StorageWriter storageWriter;
```

Comment:
Since these are constants, I think they should be in all uppercase with underscore to separate the words

Reply:
Ah yes, thank you for mentioning that :D